website/content/partials/datasource/hcp-packer-image/Config-not-required.mdx

```diff
@@ -0,0 +1,6 @@
+<!-- Code generated from the comments of the Config struct in datasource/hcp-packer-image/data.go; DO NOT EDIT MANUALLY -->
```

Comment:
This file is new, so it needs to be added to the data source main `mdx` file in order to get included in the documentation. Can you also add a similar note to the terraform one?

Reply:
@sylviamoss Ah, didn't know that, thanks! Added a note and link to this new `mdx` file.

Reply:
Validated the docs appeared on vercel deployment